Samples/WindowsCopilotRuntime/cs-wpf/WCRforWPF/Properties/launchSettings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "profiles": {
+    "WCRforWPF": {
+      "commandName": "MsixPackage",
+      "commandLineArgs": "", /* Command line arguments to pass to the app. */
```

Comment:
Is this valid json? I didn't think it supported comments natively. But you've got this working - I wonder if dotnet does some preprocessing first?

Reply:
Oh good point. I think it does but I'll remove them just for hygiene. This part will be explained in the guide anyway.